packages/zend-db/library/Zend/Db/Statement/Mysqli.php

```diff
@@ -67,7 +67,13 @@ public function _prepare($sql)
     {
         $mysqli = $this->_adapter->getConnection();
 
-        $this->_stmt = $mysqli->prepare($sql);
+        // https://www.php.net/manual/en/class.mysqli-sql-exception.php
+        // https://php.watch/versions/8.1/mysqli-error-mode
```

Comment:
I think these comments with links here are not necessary. It will be difficult to guess the context and why they are here for someone who takes a look in that part of the code in the future.
Handling exception is valid for any php version, not only 8.1+.
In my opinion it is enough to mention it in the PR description. 
Otherwise it would need additional context here and be copied to every occurrence in the code where mysqli exceptions are being handled now.